yarn-project/noir-contracts/src/contracts/noir-aztec3/src/abi.nr

```diff
@@ -95,12 +95,16 @@ struct PrivateCircuitPublicInputs {
     call_context: CallContext,
     args_hash: Field,
     return_values: [Field; crate::abi::MAX_RETURN_VALUES],
-    emitted_events: [Field; crate::abi::MAX_LOGS],
     new_commitments: [Field; crate::abi::MAX_NEW_COMMITMENTS],
     new_nullifiers: [Field; crate::abi::MAX_NEW_NULLIFIERS],
     private_call_stack: [Field; crate::abi::MAX_PRIVATE_CALL_STACK],
     public_call_stack: [Field; crate::abi::MAX_PUBLIC_CALL_STACK],
     new_l2_to_l1_msgs: [Field; crate::abi::MAX_L1_MSG_STACK],
+    // TODO #588, relevant issue: https://github.com/AztecProtocol/aztec-packages/issues/588
+    // encrypted_logs_hash: [Field; crate::abi::LOGS_HASH_SIZE],
+    // unencrypted_logs_hash: [Field; crate::abi::LOGS_HASH_SIZE],
```

Comment:
When we uncomment this, I'd explore introducing a new type like `uint256` (similar to `Point`), so it's more explicit that we're talking about a single number backed by two field elements.

Reply:
Good point. Added a comment in [b7317a2](https://github.com/AztecProtocol/aztec-packages/pull/636/commits/b7317a23fe6379033b6508964f5435c76c8bc9fa) to not forget this.